clients/admin-ui/src/features/locations/transformations.ts

```diff
@@ -73,21 +71,8 @@ export const groupByBelongsTo = (
   return byGroup;
 };
 
-export const isLocation = (data: unknown): data is Location =>
-  narrow({ selected: "boolean", regulation: ["string"] }, data);
-
-export const isRegulated = (
-  location: Location | LocationGroup,
-  locations: Location[]
-) => {
-  if (isLocation(location)) {
-    return !!location.regulation?.length;
-  }
-  const locationsInGroup = locations.filter((l) =>
-    l.belongs_to?.includes(location.id)
-  );
-  return locationsInGroup.some((l) => l.regulation?.length);
-};
+export const isRegulated = (location: Location) =>
+  !!location.regulation?.length;
```

Comment:
This utility function previously relied on the property differences between Location and LocationGroup to determine whether something was a location or a group, but now the schemas are the same.  That said, it only gets called in one place (`LocationPickerCard.tsx`) and it's on an array of definitely-Locations so we don't actually need to.